onboard vsi extension to catalog (#610)
terraform-ibm-modules/terraform-ibm-landing-zone: patterns/vsi-extension/README.md

> This architecture creates virtual server instances (VSI) for VPC in some or all of the subnets of any existing landing zone VPC deployable architecture.
> 
> NOTE: This solution only supports creating a VSI in a single VPC. If you wish to create VSIs in multiple VPCs you must create one instance of the solution per VPC.
> 

Reviewer: ```suggestion
This architecture creates virtual server instances (VSI) in some or all of the subnets of one VPC of an existing landing zone deployable architecture. To create VSIs in multiple VPCs, deploy the extension once for each VPC.
```
suggestion to pull it all together
